package.json

```diff
@@ -5,7 +5,7 @@
   "main": "diva-irma.js",
   "scripts": {
     "lint": "./node_modules/.bin/eslint -c .eslintrc *.js diva-state/*.js config/*.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "mocha '**/*.e2e.js' --exit"
```

Comment:
Maybe also run the tests in CI via the Jenkinsfile?

Reply:
Fixed